integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -226,7 +251,9 @@ def translate_queryset(self, queryset: list[Page], language_slug: str) -> None:
         asyncio.set_event_loop(loop)
 
         # Translate queryset asynchronously in parallel
-        loop.run_until_complete(self.translate_text_fields(loop, text_fields))
+        loop.run_until_complete(
+            self.translate_text_fields(loop, text_fields, output_language_level)
+        )
```

Comment:
```suggestion
        loop.run_until_complete(self.translate_text_fields(loop, text_fields))
```

Hmm, as I think about it, I guess it makes sense to place the code into the `translate_text_field()` method itself, so we don't have to carry the argument around.
Just set the additional flag right before the API call to the SUMM.AI API:

https://github.com/digitalfabrik/integreat-cms/blob/c2e9ea75443abc2b640866631e32b85b7b6e4ed5/integreat_cms/summ_ai_api/summ_ai_api_client.py#L91